content/news/034/index.md

```diff
@@ -112,6 +112,31 @@ He will be releasing videos about UI in Bevy and Rapier physics next month.
 [bevy-materials-playlist]: https://youtube.com/playlist?list=PLT_D88-MTFOMNRPAC-62Hz096aIjT4Noy
 [matthew-bryant-youtube]: https://youtube.com/channel/UC7v3YEDa603x_84PgCPytzA
 
+### [Kubecon 2022, Lightning Talk: Wasmcloud + Bevy ECS][Lightning-Talk-schedule]
+
+![A picture of speakers](./lightingtalk.jpeg)
+
+[QQparty][alanpoon-qqparty] is a serverless multiplayer game built with Bevy ECS
+and [Wasmcloud][wasmcloud_website].
+
+This month [@rustropy_gaming][alanpoon_twitter] joined Cosmonic to give
+a series of Wasmcloud talks in Kubecon 2022 VLC:
+[YouTube recording][Lightning-Talk-youtube], [slides][Lightning-Talk-pdf].
+
+Bevy can be added into Wasmcloud's actor as serverless game server.
+
+Check out Twitter [@rustropy_gaming][alanpoon_twitter]
+for announcement of QQparty going live.
+
+Deploying multiplayer game servers may be cheap in the future.
```

Comment:
Some suggested tweaks to make the text flow better:

```suggestion
This month, [@rustropy_gaming][alanpoon_twitter] joined Cosmonic to give
a series of Wasmcloud talks in Kubecon 2022 VLC. They demonstrated
how Bevy can be added into Wasmcloud's actors as a serverless
game server.

You can check out a recording of the talk on [YouTube][Lightning-Talk-youtube],
as well as viewing the [slides][Lightning-Talk-pdf]. For updates on QQparty,
follow [@rustropy_gaming][alanpoon_twitter] on Twitter!
```

Reply:
Nice! @alanpoon are you ok with the changes?

Reply:
Yes. Thanks.